packages/react-admin/src/mui/layout/DashboardMenuItem.js

```diff
@@ -16,20 +16,22 @@ const styles = {
     },
 };
 
-const DashboardMenuItem = ({ classes, onClick, translate }) => (
+const DashboardMenuItem = ({ classes, onClick, translate, options }) => (
```

Comment:
why not just `...props` ?

Reply:
I followed naming in most input components but now realized the most remaining prop name is props :)